AvgHR.py

```diff
@@ -27,6 +27,8 @@ def avghr_unbound(times, voltages):
 
 
 def avghr(times, voltages, t1=0, t2=float('inf')):
+    if times == [] or voltages == []:
+        return -1
```

Comment:
may be apt to throw an exception here if you think not satisfying this condition should stop further execution of the program